kontrakt/src/main/java/no/nav/ung/sak/kontrakt/hendelser/FødselsHendelse.java

```diff
@@ -12,13 +12,14 @@
 import jakarta.validation.constraints.NotNull;
 import no.nav.ung.kodeverk.hendelser.HendelseType;
 import no.nav.ung.sak.typer.Periode;
+import no.nav.ung.sak.typer.PersonIdent;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.NONE, getterVisibility = JsonAutoDetect.Visibility.NONE, setterVisibility = JsonAutoDetect.Visibility.NONE, isGetterVisibility = JsonAutoDetect.Visibility.NONE, creatorVisibility = JsonAutoDetect.Visibility.NONE)
 @JsonTypeName(Hendelse.FOEDSEL)
-public class FødselHendelse implements Hendelse {
+public class FødselsHendelse implements Hendelse {
```

Comment:
Denne bør hete FødselHendelse (casing som før siden det er to ord klattet sammen) eller Fødselshendelse (siden den lille s-en binder det sammen til ett ord)